stix_shifter_modules/crowdstrike/stix_translation/json/from_stix_map.json

```diff
@@ -38,7 +38,10 @@
       "parent_ref.name": ["behaviors.filename"],
       "parent_ref.command_line": ["behaviors.parent_cmdline"],
       "parent_ref.binary_ref.hashes.MD5": ["behaviors.parent_details.parent_md5"],
-      "parent_ref.binary_ref.hashes.'SHA-256'": ["behaviors.parent_details.parent_sha256"]
+      "parent_ref.binary_ref.hashes.'SHA-256'": ["behaviors.parent_details.parent_sha256"],
+      "control_graph_id":["behaviors.control_graph_i"],
```

Comment:
None of these are standard properties for the STIX process object. These would need to go as custom properties. 
https://docs.oasis-open.org/cti/stix/v2.0/cs01/part4-cyber-observable-objects/stix-v2.0-cs01-part4-cyber-observable-objects.html#_Toc496716272